pkg/services/ngalert/metrics/multi_org_alertmanager.go

```diff
@@ -1,21 +1,30 @@
 package metrics
 
 import (
+	"fmt"
+	"strconv"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 )
 
 type MultiOrgAlertmanager struct {
-	Registerer               prometheus.Registerer
+	Registerer prometheus.Registerer
+	registries *UserRegistries
+
 	ActiveConfigurations     prometheus.Gauge
 	DiscoveredConfigurations prometheus.Gauge
-	registries               *OrgRegistries
+
+	aggregatedMetrics *AlertmanagerAggregatedMetrics
```

Comment:
You're probably wondering where is the test for this - well, it's complicated.

To do the testing, I'd have to basically "copy" all of the metrics from the Alertmanager just for the test setup - it feels like we're entering the land of diminishing returns and have opted out from doing anything like that right now.

If you feel strongly about having them I can probably figure something out at the expense of more time.